zebra/rtadv.c

```diff
@@ -1756,6 +1841,17 @@ DEFPY_YANG (ipv6_nd_managed_config_flag,
 	return nb_cli_apply_changes(vty, NULL);
 }
 
+void lib_interface_zebra_ipv6_router_advertisements_managed_flag_cli_write(
+	struct vty *vty, const struct lyd_node *dnode, bool show_defaults)
+{
+	bool managed_flag = yang_dnode_get_bool(dnode, NULL);
+
+	if (managed_flag)
+		vty_out(vty, " ipv6 nd managed-config-flag\n");
+	else if (show_defaults)
```

Comment:
some commands like this, we show defaults, but  some don't, any reason?

Reply:
It depends on whether the leaf has the `default` statement in the model. If there's no `default` statement, then the leaf has two states - `set` or `not-set`. If it's not set, the `cli_show` callback won't be called and we don't need to check the `show_default` flag. But it the leaf has the `default` statement, then it basically has three states - `set`, `not-set`, or `set-to-default`. For the latter, the `cli_show` callback will be triggered, so we need to check and show the value that is equal to the default one only when the `show_default` flag is `true`.